service/src/main/java/bio/terra/policy/app/controller/TpsApiController.java

```diff
@@ -46,6 +49,14 @@ public ResponseEntity<Void> deletePao(UUID objectId) {
     return new ResponseEntity<>(HttpStatus.NO_CONTENT);
   }
 
+  @Override
+  public ResponseEntity<ApiTpsPaoGetResult> evaluateRegionDatacenter(
+      UUID objectId, String datacenter, String platform) {
+    Pao pao = paoService.getPao(objectId);
+    regionService.isDatacenterAllowedByPao(pao, datacenter, platform);
```

Comment:
nit: reading that method name, I would expect it to return a boolean. Maybe `validateDatacenterAllowed`?